packages/react-components/react-tree/library/src/components/Tree/Tree.types.ts

```diff
@@ -20,6 +20,13 @@ export type TreeNavigationData_unstable = {
   value: TreeItemValue;
   parentValue: TreeItemValue | undefined;
 } & (
+  | { event: React.FocusEvent<HTMLElement>; type: 'Focus' }
```

Comment:
isn't this a breaking change?

Reply:
Yes, it is. Any suggestions to avoid it?